packages/database/data-migrations/20240318050527_add_languages_and_survey_languages/data-migration-welcomeCard-fix.ts

```diff
@@ -0,0 +1,50 @@
+// migration script to add empty strings to welcome card headline in default language, if it does not exist
+// WelcomeCard.headline = {} -> WelcomeCard.headline = {"default":""}
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+async function main() {
+  await prisma.$transaction(
+    async (tx) => {
+      const surveys = await tx.survey.findMany({
+        select: {
+          id: true,
+          welcomeCard: true,
+        },
+      });
+
+      if (surveys.length === 0) {
+        // stop the migration if there are no surveys
+        return;
+      }
+      let count = 0;
+      for (const survey of surveys) {
+        const updatedSurvey = structuredClone(survey) as any;
+        if (
+          updatedSurvey.welcomeCard &&
+          updatedSurvey.welcomeCard.headline &&
```

Comment:
Can we do are more explicit check here and check if it's an empty object?
If we miss something else we might overwrite an existing headline value with an empty value and loose information.

Reply:
Added `Object.keys(updatedSurvey.welcomeCard.headline).length === 0`, to check if headline is an empty object